backend/apps/owasp/api/search/chapter.py

```
@@ -1,11 +1,17 @@
 
-from algoliasearch_django import raw_search
-
-from apps.owasp.models.chapter import Chapter
-
-
-def get_chapters(query, attributes=None, limit=25, page=1, searchable_attributes=None):
+from apps.common.typesense import Typesense
+from apps.core.utils.params_mapping_typesense import get_typesense_params_for_index
+
+
+def get_chapters(
```

Ruff (0.8.2)

Missing argument description in the docstring for `get_chapters`: `searchable_attributes_weights`

(D417)